tests/EvergreenLogListener.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace MongoDB\Tests;
+
+use Exception;
+use PHPUnit\Framework\AssertionFailedError;
+use PHPUnit\Framework\SelfDescribing;
+use PHPUnit\Framework\Test;
+use PHPUnit\Framework\TestFailure;
+use PHPUnit\Framework\TestListener;
+use PHPUnit\Framework\TestSuite;
+use PHPUnit\Framework\Warning;
+use PHPUnit\Util\Filter;
+use PHPUnit\Util\Printer;
+use function get_class;
+use function json_encode;
+use function round;
+
+// phpcs:disable SlevomatCodingStandard.Exceptions.ReferenceThrowableOnly.ReferencedGeneralException
+class EvergreenLogListener extends Printer implements TestListener
```

Comment:
Since we already use [symfony/phpunit-bridge](https://github.com/symfony/phpunit-bridge), I noticed it has an alias for a SymfonyTestsListener interface. I think you could leverage that instead of implementing PHPUnit's interface directly and preserve our ability to test with PHPUnit 8.x.

Reply:
Unfortunately, that class is not a generic implementation for test listeners, but rather a specific one to support the use-case of the PHPUnit bridge.

Reply:
Ah, I just poked my head into SymfonyTestsListenerTrait.

![](https://media.giphy.com/media/3ohs7KViF6rA4aan5u/giphy.gif)